hooks/playbooks/control_plane_ceph_backends.yml

```diff
@@ -108,6 +108,12 @@
                 path: /spec/manila/enabled
                 value: {{ cifmw_services_manila_enabled | default('false') }}
 
+              - op: add
```

Comment:
shouldn't this be applied only when cifmw_services_manila_enabled is true? Or given the fact that the definition of manilaAPI is always there it doesn't matter?

Reply:
exactly, it doesn't matter, and it can simplify any potential customization in a zuul job (where we have the distinction between `native_cephfs` and `cephnfs`) because the protocols are already configured in the API in case Ceph is selected as a backend.